feature_engine/transformation/yeojohnson.py

```diff
@@ -152,3 +152,45 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
             X[feature] = stats.yeojohnson(X[feature], lmbda=self.lambda_dict_[feature])
 
         return X
+
+    def inverse_transform(self, X: pd.DataFrame) -> pd.DataFrame:
+        """
+        Apply the inverse-transform function Yeo-Johnson.
```

Comment:
All inverse_transform methods take the same info: "Convert the data back to the original representation." Could you please update this sentence?